src/odin/commands/configure.rs

```diff
@@ -118,12 +123,12 @@ impl Configuration {
 
   fn perform_preflight_checks(&self) -> Result<(), Box<dyn std::error::Error>> {
     let paths_to_check = vec![
-      Path::new("/home/steam/valheim"),
-      Path::new("/home/steam/scripts"),
-      Path::new("/home/steam/.bashrc"),
+      PathBuf::from_iter([&self.steam_home_dirname, "valheim"]),
+      PathBuf::from_iter([&self.steam_home_dirname, "scripts"]),
+      PathBuf::from_iter([&self.steam_home_dirname, ".bashrc"])
```

Comment:
my attempts to support running external of the container, by de-hardcoding these values